### DISCUSSION TITLE
How to get the result of an event?

### DISCUSSION THREAD

**Discussion by tsouras** [cloudbet/docs#19] [General]
Let's say that a soccer match has finished.
How can I get the result of the match ?

**Comment by kgravenreuth** [comment]
Hi @tsouras 

The Cloudbet API currently doesn't provide results of ended matches yet. You can, however, get a [history of your placed bets](https://www.cloudbet.com/api/?utm_source=github.com&utm_medium=community&utm_campaign=axis&urls.primaryName=Trading#/Trading/BetHistory) through the Trading API so that you know how your bet was settled by our system. 

You can find a walkthrough on how to use this endpoint (and others) in our [Betting API tutorial blog post](https://www.cloudbet.com/en/blog/posts/sports-betting-api-tutorial-with-golang?utm_source=github.com&utm_medium=community&utm_campaign=axis#toc-8)

**Reply by ktalovmorx** [reply]
Is very important for the gamblers. Sometimes we simply needs to know the result score of an event.

**Reply by kgravenreuth** [reply]
Thanks for the feedback @ktalovmorx . We will consider looking into results for events.

**Reply by woutervermeersch** [reply]
Hello, I was also searching for that peace of critical functionality within the docs.. 

Unfortunate to see that it isn’t implemented yet… 

This thrives me to consider another api in order to get the full package… 

Working with two separate apis would not only lead to inconsistency  within the pairing of event ids but lead to much harder maintenance and setup costs. 

I am glad to see you guys at cloudbet are looking for a workaround.. what’s the ETA on the project?

**Comment by ktalovmorx** [comment]
Maybe adding the final score here:

"status": "RESULTED",
  "markets": {},
  "name": "Burnley V Liverpool",
  "key": "c45e8-burnley-fc-v-c5906-liverpool-fc",
  "cutoffTime": "2022-02-13T14:00:00Z",
  "metadata": {
    "opinion": [
      {
        "outcome": "away",
        "probability": 0.7875389,
        "marketKey": "soccer.match_odds",
        "params": ""
      },
      {
        "outcome": "draw",
        "probability": 0.08420649,
        "marketKey": "soccer.match_odds",
        "params": ""
      },
      {
        "outcome": "home",
        "probability": 0.12825459,
        "marketKey": "soccer.match_odds",
        "params": ""
      }
    ]
  }

**Reply by woutervermeersch** [reply]
This seems also for me the perfect place to put the final score in. Actually already searched the api in that direction but unfortunately got the same poor skimmed down result back from the endpoint telling me nothing I didn’t know from that particular event id

**Reply by ktalovmorx** [reply]
Exactly, so they would not have to create another endpoint but simply add that field.

**Comment by kgravenreuth** [comment]
@ktalovmorx @woutervermeersch Thanks for your suggestions folks. We've considered your feedback and currently, we can provide you information about how your API bets have resulted via the [Bet History](https://www.cloudbet.com/api/?utm_source=github.com&utm_medium=community&utm_campaign=axis&urls.primaryName=Trading#/Trading/BetHistory) endpoint of our Trading API. 

For example, If you'd like to know the result of a given event, then you can place bets on the Winner or Moneyline markets for that event and then check the bet results to know the event's results.

We are currently not planning to add results to the Feed API, while we thank you for your suggestions above.
